src/models/tick-model/index.js

```diff
@@ -45,16 +44,27 @@ export default function createTickModel({
     return { min, max, explicitType, distance, size, measure };
   }
 
-  const formatters = {
-    x: chartModel.query.getFormatter(KEYS.FIELDS.X),
-    y: chartModel.query.getFormatter(KEYS.FIELDS.Y),
+  const formatPatterns = {
+    [KEYS.SCALE.X]: undefined,
+    [KEYS.SCALE.Y]: undefined,
   };
 
   function resolve(axis, prop) {
+    if (!formatPatterns[axis]) {
+      const field = axis === KEYS.SCALE.X ? 'qMeasureInfo.0' : 'qMeasureInfo.1';
```

Comment:
maybe
```suggestion
      const field = axis === KEYS.SCALE.X ? KEYS.FIELDS.X : KEYS.FIELDS.Y;
```

Reply:
I thought about that. The problem is `KEYS.FIELDS.X = 'qMeasureInfo/0'`, while for layoutService, we need to call: `layoutService.getHyperCubeValue('qMeasureInfo.0.qIsAutoFormat, false);` (with a dot, not a slash)